### ISSUE TITLE
Possible improved way to check whether postcodes are eligible or not

### ISSUE THREAD

**Issue by harryvos** [alphagov/govuk-shielded-vulnerable-people-service#173]
In case you’ve not yet seen it, I thought I’d share the latest release of the updated [National Statistics Unique Property Reference Number Lookup (NSUL)](https://geoportal.statistics.gov.uk/datasets/74586dfe01c14617999c558c1d5e0647)

According to [Owen Boswarva’s tweet](https://twitter.com/owenboswarva/status/1328309300721283073?s=21), “That means we can derive extents of OS Code-Point Open postcodes, as #opendata, and link UPRNs to many other datasets.”

Would this help check eligibility more accurately?

**Comment by iainboyd** [comment]
Yes! Thanks Harry, this would be a more precise way of both identifying addresses and mapping to alert levels and local authorities. The data gives a specific UPRN with accompanying full postcode and local authority code.
